src/scripts/main.js

```diff
@@ -1,3 +1,49 @@
 'use strict';
 
 // write code here
+const thead = document.querySelector('thead');
+const th = thead.children[0].children;
```

Comment:
Ensure that the `thead` element has children and the first child has the expected structure. If the table structure changes, this line might throw an error.